doc/concepts/lif_cell.rst

```diff
@@ -21,20 +23,19 @@ mechanisms.
 
 The LIF cell's time dynamics are this:
 
-0. :math:`U_\mathrm{m}(0) = E_\mathrm{L}`
-1. If the cell is in its refractory state :math:`U_\mathrm{m}(t) = E_\mathrm{L}`
-2. Otherwise :math:`U'_\mathrm{m}(t) = \sum\limits_\mathrm{spike} w_\mathrm{spike} \cdot\delta(t - t_\mathrm{spike}) -\frac{1}{\tau_\mathrm{m}}U_\mathrm{m}(t)`
+0. :math:`U_\mathrm{m}(0) = V_\mathrm{0}`,
+1. If the cell is in its refractory state :math:`U_\mathrm{m}(t) = E_\mathrm{R}`
+2. Otherwise :math:`U'_\mathrm{m}(t) = \sum\limits_\mathrm{spike} w_\mathrm{spike} \cdot\delta(t - t_\mathrm{spike}) - \frac{1}{\tau_\mathrm{m}}\left(U_\mathrm{m}(t)) - E_\mathrm{L}\right)`
```

Comment:
`w_\mathrm{spike}` is not defined above.

Reply:
Yes, it's not part of the cell, but it should still be defined somewhere.

Reply:
There, it is now explained. I added C++ API docs for LIF as well.